manifests/check.pp

```diff
@@ -162,6 +162,22 @@
     }
   }
 
+  case $handlers {
+    'absent': { $handlers_array = undef } 
+    String:   { $handlers_array = [ $handlers ] ; deprecation('handlers', 'Parameter handlers should be an array') } 
```

Comment:
syntax error here found by Travis

Reply:
I see. And I find it disturbing, as on my devel environment I don't see errors.
Going to try alternatives.